sqlite-android/src/main/java/io/requery/android/database/sqlite/SQLiteStatementType.java

```diff
@@ -64,34 +64,31 @@ public static int getSqlStatementType(String sql) {
         if (sql.length() < 3) {
             return STATEMENT_OTHER;
         }
-        String prefixSql = sql.substring(0, 3).toUpperCase(Locale.US);
-        switch (prefixSql) {
-            case "SEL":
-                return STATEMENT_SELECT;
-            case "INS":
-            case "UPD":
-            case "REP":
-            case "DEL":
-                return STATEMENT_UPDATE;
-            case "ATT":
-                return STATEMENT_ATTACH;
-            case "COM":
-                return STATEMENT_COMMIT;
-            case "END":
-                return STATEMENT_COMMIT;
-            case "ROL":
-                return STATEMENT_ABORT;
-            case "BEG":
-                return STATEMENT_BEGIN;
-            case "PRA":
-                return STATEMENT_PRAGMA;
-            case "CRE":
-            case "DRO":
-            case "ALT":
-                return STATEMENT_DDL;
-            case "ANA":
-            case "DET":
-                return STATEMENT_UNPREPARED;
+        String prefixSql = sql.substring(0, 3).toUpperCase(Locale.ROOT);
```

Comment:
This seems like it might be a semantic change - from Locale.US to Locale.ROOT - what will it effect?

In general the switch seemed so much more terse, and yet still understandable, why the move to if/elseif?

Reply:
Sorry, I closed PR because of this. Found it right after.
Code must be completely not like that. Fix in some minutes